fzf-pass

```diff
@@ -1,5 +1,17 @@
 #!/usr/bin/env bash
 
+
+function ldotool {
+    ydotool type ""
+    if [ $? eq 0 ]; then
+        ydotool "$@"
+    else
+        xdotool "$@"
```

Comment:
I don't have xdotool installed on my machine so I get this:
```
./fzf-pass: line 9: xdotool: command not found
```
This is probably related to the error on line 6, see my comment there.